shipitsteven/note-app
# Added electron to the app

## Description of Issue
Added Electron wrapper to our app. 
You can now run it as a desktop app after `yarn install` and `yarn electron-dev`

This addition only includes scripts/dependencies to run the app in a development environment, build scripts and builder will be added in the future.

## Testing
*Have you tested this change? On what devices?*
Windows 10

### To Test
- Navigate to the client folder in a terminal
- run `yarn install`
- run `yarn electron-dev` or `yarn run electron-dev`
- App should spin up as a local process

_Note:_ links are broken in preview (it disappears after you click it)

## Time Spent
- *How long did you estimate this task would take?* 2 hours
- *How long did it take you to work on this issue?* 2 hours

## Checklist
